jujugui/static/gui/src/app/components/boolean-config/_boolean-config.scss

```diff
@@ -4,13 +4,18 @@
     position: relative;
 
     &--title {
-      padding-top: 8px;
+        margin: auto;
+        margin-left: 0;
     }
 
     &--toggle {
-      position:absolute;
-      right: 0;
-      top: 0;
+        margin: auto;
+        margin-left: 4px;
+        margin-right: 0;
```

Comment:
Again instead of overriding the top styling this would be better set once as `margin: auto 4px auto 0`